### DISCUSSION TITLE
libvips support for `lossless_jpg` cjxl option

### DISCUSSION THREAD

**Discussion by Qhilm** [libvips/libvips#4213] [Q&A]
`vips jxlsave a.jpg b.jxl` results in significantly larger files, compared to cjxl.

I assume it's because libvips does not use `lossless_jpeg` here, but wondering if this option exists. I could not find it mentioned so far.

**Comment by jcupitt** [comment]
Hello, libvips always decodes and recodes, so `jxlsave` doesn't have direct access to the source image. This kind of file-format hackery is out of scope.

**Comment by Qhilm** [comment]
Thought so, thanks for confirming.
